tftrt/examples/cpp/image-classification/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Suggestions:
- Divide into sections like below
- Add cache variables for things like Tensorflow source/install/python locations, TRT install paths

Tensorflow build directly copies includes for CUDA and TensorRT repos without reformatting, so you should be able to use `FindCUDAToolkit` to bring in `CUDA::cudart` target for CUDA includes and `find_path(... NvInfer.h)` to find TensorRT includes from user-specified base directory.


```
cmake_minimum_required(VERSION 3.13)
project(TensorRTExample)

#-------------------------------------------------------------
# Configuration
#-------------------------------------------------------------
set(TF_SOURCE_DIR "/some/default" CACHE PATH "Path to Tensorflow source directory")
set(TF_INSTALL_DIR "/some/default" CACHE PATH "Path to Tensorflow install directory")

#-------------------------------------------------------------
# Dependencies
#-------------------------------------------------------------
find_package(CUDAToolkit REQUIRED) # brings in CUDA::cudart
find_library(tf_shared NAMES libtensorflow.so HINTS ${TF_INSTALL_DIR}) 
etc....

#-----------------------------------------------------------
# Example Targets
#-----------------------------------------------------------
etc... 
```